src/assets/js/global.js

```diff
@@ -1,7 +1,20 @@
-/* global jQuery */
+/* global jQuery, Modernizr, Pikaday, GT */
```

Comment:
We can remove `Pikaday, GT` from globals. We should probably use a **.jshintrc** file if we go this route of using the module loader.